dist/uploadcare-netlifycms.js

```diff
@@ -0,0 +1 @@
transform:void 0,insertInto:void 0};n(11)(r,o);r.locals&&(e.exports=r.locals)},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0}),t.Control=void 0;var r=i(n(0));n(14);var o=i(n(9));function i(e){return e&&e.__esModule?e:{default:e}}function u(e){return(u="function"==typeof Symbol&&"symbol"==typeof Symbol.iterator?function(e){return typeof e}:function(e){return e&&"function"==typeof Symbol&&e.constructor===Symbol&&e!==Symbol.prototype?"symbol":typeof e})(e)}function a(e,t){return(a=Object.setPrototypeOf||function(e,t){return e.__proto__=t,e})(e,t)}function c(e,t){for(var n=0;n<t.length;n++){var r=t[n];r.enumerable=r.enumerable||!1,r.configurable=!0,"value"in r&&(r.writable=!0),Object.defineProperty(e,r.key,r)}}function f(e,t){return!t||"object"!==u(t)&&"function"!=typeof t?function(e){if(void 0===e)throw new ReferenceError("this hasn't been initialised - super() hasn't been called");return e}(e):t}function l(e){return(l=Object.getPrototypeOf||function(e){return e.__proto__})(e)}var s=function(e){function t(){return function(e,t){if(!(e instanceof t))throw new TypeError("Cannot call a class as a function")}(this,t),f(this,l(t).apply(this,arguments))}var n,i,u;return n=t,(i=[{key:"onUploadComplete",value:function(e){this.props.onChange(e)}},{key:"render",value:function(){var e=this.props.field.get("options").toJS(),t=this.props.value,n=t&&t.cdnUrl;return r.default.createElement("div",{className:"nc-controlPane-widget nc-imageControl-imageUpload"},r.default.createElement("span",{className:"nc-imageControl-message"},r.default.createElement(o.default,{options:e,value:n,onUploadComplete:this.onUploadComplete.bind(this)})))}}])&&c(n.prototype,i),u&&c(n,u),function(e,t){if("function"!=typeof t&&null!==t)throw new TypeError("Super expression must either be null or a function");a(e.prototype,t&&t.prototype),t&&a(e,t)}(t,e),t}(r.default.Component);t.Control=s},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0}),t.default=function(e){var t=r.default.merge(s,e);(0,c.initializeUploadcare)(t).then(function(){return function(e){i.register();var t=o.default.getEditorComponents();t._root.entries=[],t.size=0,o.default.registerEditorComponent((0,u.createSingle)(e)),o.default.registerEditorComponent((0,a.createMultiple)(e))}(t)})};var r=f(n(1)),o=f(n(3)),i=function(e){if(e&&e.__esModule)return e;var t={};if(null!=e)for(var n in e)if(Object.prototype.hasOwnProperty.call(e,n)){var r=Object.defineProperty&&Object.getOwnPropertyDescriptor?Object.getOwnPropertyDescriptor(e,n):{};r.get||r.set?Object.defineProperty(t,n,r):t[n]=e[n]}return t.default=e,t}(n(2)),u=n(7),a=n(6),c=n(5);function f(e){return e&&e.__esModule?e:{default:e}}var l,s={effectsTab:!0,widgetOptions:{publicKey:"demopublickey",integration:(l=function(){try{var e=window.initCMS.toString().match(/\"Netlify CMS version ([\d\.]*)\"/m);return e?e[1]:"unknown"}catch(e){return"unknown"}}(),"NetlifyCMS/".concat(l,"; Uploadcare-NetlifyCMS/").concat("1.0.0"))}}}])});
```

Comment:
I think we don't need to save `dist` to git O_o

Reply:
@Zmoki I was going to remove it when npm package will be published. It is needed for demo.